spec/mail/mailbox.mdown

```diff
@@ -160,6 +160,17 @@ Mailboxes can be created, updated and destroyed using the *setMailboxes* method.
   A map of mailbox id to objects containing the properties to update for that Mailbox. If `null`, no objects will be updated.
 - **destroy**: `String[]|null`
   A list of ids for Mailboxes to permanently delete. If `null`, no objects will be deleted.
+- **onLastReferenceMoveTo**: `String|null`
+  A message must be in at least one mailbox at all times. Destroying mailboxes will remove any messages in them from that mailbox. If the message is not in any other mailboxes, it will be added to the mailbox with the id given by this argument, if not `null`. If `null`, the message will be destroyed.
```

Comment:
A remark triggered internally, regarding JMAP philosophy by https://github.com/fvn-linagora :

```
not providing a value to onLastReferenceMoveTo (eqv to set as null) should NOT trigger any destructive operation.
```

That is, the mailbox SHOULD NOT be removed and the operation SHOULD be rejected with a `mailboxHasMessage` error.
```